apps/src/templates/courseOverview/ResourcesEditor.js

```diff
@@ -73,23 +97,21 @@ export default class ResourcesEditor extends Component {
         )}
 
         <div style={styles.box}>
+          <div style={styles.error}>{errorString}</div>
           <div style={{marginBottom: 5}}>Preview:</div>
-          <CourseOverviewTopRow
-            sectionsInfo={[]}
-            id={-1}
-            title="Unused title"
-            resources={resources.filter(x => !!x.type)}
-          />
+          {this.props.renderPreview(resources.filter(x => !!x.type))}
```

Comment:
I kind of like the first way because it shows all the UI together, but I'm not sure it matters much either.